packages/analytics-js-integrations/src/integrations/MicrosoftClarity/browser.js

```diff
@@ -55,7 +56,13 @@ class MicrosoftClarity {
     if (context?.traits?.customPageId) {
       customPageId = context.traits.customPageId;
     }
-    window.clarity('identify', userId, sessionId, customPageId);
+
+    const identifyPromise = window.clarity('identify', userId, sessionId, customPageId);
+    if (typeof identifyPromise?.then === 'function') { // Clarity SDK is ready
```

Comment:
this is fine but just a suggestion:
https://stackoverflow.com/a/27760489/18047180

Reply:
Feel free to ignore

Reply:
Ignoring for now